public/app/features/dashboard-scene/scene/DashboardScene.test.tsx

```diff
@@ -905,8 +906,229 @@ describe('DashboardScene', () => {
       expect(scene.managedResourceCannotBeEdited()).toBe(false);
     });
   });
+
+  describe('getExpressionTypes', () => {
+    it('should extract SQL expression type from V1 and V2 dashboards', () => {
+      const scene = buildTestScene();
+      const saveModel = createV1DashboardWithExpressions(['sql']);
+      const saveModelV2 = createV2DashboardWithExpressions(['sql']);
+
+      const result = scene.getExpressionTypes(saveModel);
+      const resultV2 = scene.getExpressionTypes(saveModelV2);
+
+      expect(result).toEqual(['sql']);
+      expect(resultV2).toEqual(['sql']);
```

Comment:
No big deal at all really, but I would normally lean towards writing these as separate tests. Not sure if you share that feeling at all? 
But please don't bother changing this again, even if you agree - not worth the effort.